cadl-tests/src/main/java/com/basicpolymorphicmodels/BasicPolymorphicModelsAsyncClient.java

```diff
@@ -107,11 +107,8 @@ public Mono<Response<BinaryData>> setValueWithPolymorphicPropertyWithResponse(
         return this.serviceClient.setValueWithPolymorphicPropertyWithResponseAsync(input, requestOptions);
     }
 
-    /*
-     * Generated convenience method for setValueWithResponse
-     */
     /**
-     * The setValue operation.
+     * Generated convenience method for setValueWithResponse. The setValue operation.
```

Comment:
In code, we put a new line after 'Generated convenience method for setValueWithResponse.'. 
https://github.com/Azure/autorest.java/blob/f2e074a42ed435cbe58daaff719ce2085bbf2b6f/javagen/src/main/java/com/azure/autorest/template/ConvenienceMethodTemplateBase.java#L49
But after formatting, `Generated convenience method for setValueWithResponse.` and `The setValue operation.` appear in the same line. Please let me know if there is any concern.